src/Operation/CreateEncryptedCollection.php

```diff
@@ -97,21 +97,21 @@ public function __construct(private string $databaseName, private string $collec
      * "encryptedFields" option and reconstruct the internal CreateCollection
      * operation used for creating the encrypted collection.
      *
-     * The $encryptedFields reference parameter may be used to determine which
-     * data keys have been created.
+     * Returns the data keys that have been created.
      *
      * @see \MongoDB\Database::createEncryptedCollection()
      * @see https://www.php.net/manual/en/mongodb-driver-clientencryption.createdatakey.php
+     * @return array
      * @throws DriverRuntimeException for errors creating a data key
      */
-    public function createDataKeys(ClientEncryption $clientEncryption, string $kmsProvider, ?array $masterKey, ?array &$encryptedFields = null): void
+    public function createDataKeys(ClientEncryption $clientEncryption, string $kmsProvider, ?array $masterKey): array
```

Comment:
The function return null, we don't need to have a by-reference parameter to output the data keys. Using the returned value is more common.
This change can be backported to 1.x, with a deprecation message if `$encryptedFields` parameter is passed.

Reply:
> This change can be backported to 1.x

FWIW, I'm OK skipping that and just communicating this in the upgrade notes. I think it's very unlikely that anyone was actually using this API.

Defer to you, though.

Reply:
If we consider the operation (and thus the API) internal in 1.x, I agree with not adding a deprecation layer in 1.x.